components/ComplexInput.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, HTMLProps, useEffect, useState } from "react";
+
+interface ComplexInputProps<T>
```

Comment:
This is actually unused now, but i'm kinda tempted just to leave it in cause i think it's powerful. I was using it for manual array input in my first iteration.

Reply:
cool with leavin this in!